ekip/everykid/templates/get-your-pass/educator_passes.html

```diff
@@ -8,17 +8,19 @@
 
 {% block content %}
 
-  <h2>Get Vouchers For Your Students</h2>
+  <h1>Get vouchers<br/>for your students</h1>
 
   <p>As a 4th grade educator, you can download a learning activity and print paper vouchers for your students. These vouchers will give them free access for up to a full year to all federal lands and waters.</p>
 
   <div class="explanation">
 
     <h3>You are a 4th-grade educator if ...</h3>
-    <p>You are a teacher of 4th graders at a US school or a US military school, or</p>
```

Comment:
The US military school bit needs to be there. 

In general, the content I add is based on the what's been agreed upon.